Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/EntryDelegate.cpp

```diff
@@ -62,9 +62,14 @@ namespace AzToolsFramework
             QWidget* widget = QStyledItemDelegate::createEditor(parent, option, index);
             if (auto* lineEdit = qobject_cast<QLineEdit*>(widget))
             {
-                connect(lineEdit,&QLineEdit::editingFinished, this, [&, lineEdit]()
+                connect(lineEdit, &QLineEdit::editingFinished,
+                    this, [this]()
                     {
-                        Q_EMIT RenameEntry(lineEdit->text());
+                        auto sendingLineEdit = qobject_cast<QLineEdit*>(sender());
+                        if (sendingLineEdit)
```

Comment:
you can make this apart of the if statment
```
if (auto* sendingLineEdit = qobject_cast<QLineEdit*>(sender()))
```

Reply:
I originally left it out of the statement to make debugging easier. I think we can leave it out for now since jjjoness might want to investigate the rename not working.